monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -82,7 +82,8 @@ sealed trait Rx[+A] { self =>
   /**
    * Produces a `Rx` containing cumulative results of applying a binary
    * operator to each element of this `Rx`, starting from a `seed` and the
-   * current value, and moving forward in time.
+   * current value of the upstream `Rx`, and moving forward in time; no internal
+   * state is maintained.
    *
    * ```
    * val numbers: Rx[Int]
```

Comment:
The line below this one still has the `fold` instead of `foldp` typo :)

Reply:
oops, fixed